app/src/main/java/org/pursuit/usolo/map/MapFragment.java

```diff
@@ -179,6 +184,23 @@ private void addZoneMarker(Style style, LatLng zone) {
 
     }
 
+    private void showZoneDialog() {
```

Comment:
consider delegating this method to the HostActivity so that we aren't inflating a fragment from within another fragment